### PR TITLE
feat: add color style 

### PR DESCRIPTION
**Pull request by HrRn**
## Overview

I added color style.

## Changes

- [x] Add colors

## Assignee Checklist:

- [x] The base branch is correct (no accidental merges)
- [x] The branch name follows our branch naming rules
- [x] The PR title follows our PR title rules
- [x] My code follows our coding style
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors in the console

## Reviewer Checklist:

- [x] Can use classes correctly

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -23,6 +23,8 @@ module.exports = {
         accent: {
           DEFAULT: 'rgb(var(--color-accent))',
         },
+        black: 'rgb(var(--color-black))',
+        overlay: 'rgb(var(--color-overlay))',
```
Comment (KaiKoide) [inline tailwind.config.ts:27]: What was the purpose behind adding the overlay?

Comment (HrRn) [reply tailwind.config.ts:27]: The background overlay when the link is clicked and opens the Drawer.

Comment (KaiKoide) [reply tailwind.config.ts:27]: I understand! Thank you for your answer!

**Review by KaiKoide — APPROVED** [review]
LGTM!

**Pull request merged**